batch/kueue-intro/team-a-job.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START gke_batch_kueue_intro_team_a_job]
+apiVersion: batch/v1
+kind: Job
+metadata:
+  namespace: team-a # Job under team-a namespace
+  generateName: sample-job-
+  annotations:
+    kueue.x-k8s.io/queue-name: team-a-lq # Point to the LocalQueue
+spec:
+  ttlSecondsAfterFinished: 60 # Job will be deleted after 60 seconds
+  parallelism: 3 # This Job will have 3 replicas running at the same time
+  completions: 3 # This Job requires 3 completions
+  suspend: true # Set to true and allow Kueue to control the Job
+  template:
+    spec:
+      nodeSelector:
+        cloud.google.com/gke-accelerator: "nvidia-tesla-t4" # Specify the GPU hardware
+      containers:
+      - name: dummy-job
+        image: gcr.io/k8s-staging-perf-tests/sleep:latest
+        args: ["10s"] # Sleep for 10 seconds
+        resources:
+          requests:
+            cpu: "50m"
+            memory: "100Mi"
+            ephemeral-storage: "100Mi"
+            nvidia.com/gpu: "1"
+          limits:
+            nvidia.com/gpu: "1"
```

Comment:
```suggestion
          limits:
            cpu: "500m"
            memory: "512Mi"
            ephemeral-storage: "512Mi"
            nvidia.com/gpu: "1"
```

Reply:
Eliminates

Warning: Autopilot increased resource requests for Job team-b/sample-job-* to meet requirements. See http://g.co/gke/autopilot-resources

message